test/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+bundle exec kitchen create
```

Comment:
This script does not explicitly run `bundle install`, and will fail by default:

```
╰─ make test
Could not find aws-sdk-core-2.11.154 in any of the sources
Run `bundle install` to install missing gems.
make: *** [test] Error 7
```

Reply:
Good catch. Fixed by adding `bundle install` in this file. https://github.com/terraform-google-modules/terraform-google-pubsub/pull/1/commits/6aa8337c35e5769c42255ba0a3a50268f1baa57a